00-Introducing-HOOMD-blue/06-Equilibrating-the-System.ipynb

```diff
@@ -32,6 +32,20 @@
     "import math"
```

Comment:
<p>Should we be referring to this as a GSD Analyzer, or a GSD writer? My vote is for writer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>